modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/modes/Table.tsx

```diff
@@ -473,6 +476,80 @@ function Table({
 		});
 	};
 
+	const saveFDSFields = async ({
```

Comment:
This is the largest change in PR. `AddFieldsModalContent` contained the logic specific to table VM fields saving. Now we are using the common `FieldSelectModalContent` selection component, and the specific logic is in the parent `Table` component.